src/mono/netcore/CoreFX.issues_linux.rsp

```diff
@@ -21,6 +21,9 @@
 # StackOverflow somewhere here
 -nonamespace System.Threading.Tasks.Tests
 
+# Stack overflow, might need tuning or reduce stack space used compared to CoreCLR
+-nomethod  Microsoft.Extensions.DependencyInjection.Tests.ServiceProviderCompilationTest.CompilesInLimitedStackSpace
+
```

Comment:
Should we have used ActiveIssue instead?
```
[ActiveIssue("https://github.com/dotnet/runtime/issues/33894", TestRuntimes.Mono)]
```

Reply:
We could do that, I was trying to find out how to suppress for mono, if we have an attribute that is better.

Reply:
We are still in the process of removing what we skip from rsp files. We're pretty much down to linux & windows.  Attributes like `[ActiveIssue]` are the way to go.